pkg/runtime/runtime.go

```diff
@@ -66,12 +62,5 @@ func Start(syncr sync.ISync, server service.IService, ctx context.Context) {
 		}
 	}()
 
-	go server.Serve(
-		func(ir service.IServiceRequest) service.IServiceResponse {
-			if ir.GetRequestType() == service.SERVICE_REQUEST_ALL_FLAGS {
-				return ir.GenerateServiceResponse(getSyncPayload())
-			}
-			return nil
-		})
-
+	go server.Serve(ev)
```

Comment:
Server has handlers of some shape, which are run with the `.Serve()` method, which takes the `IEvaluation` for the handlers to call.